src/index.html

```diff
@@ -9,6 +9,88 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+
+    <header class="header">
+
+      <a class="header_logo-link" href="#home">
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="logo"
```

Comment:
Alt attribute value should be maximally clear and maximally descriptive. That's an alternate description of the image, so I would come up with sth like that:
```suggestion
          alt="MOYO logo"
```